### DISCUSSION TITLE
Payments for M0.T1 — Design UI, technical architecture

### DISCUSSION THREAD

**Discussion by Zk2u** [syvita/daoos#12] [Payments]
# Task 1 — Design UI, technical architecture

Task 1 is allocated $2500 to be split.

|GH Username |Worked on M0T1?|What they did | Percentage |
|------------|---------------|-----------------------------------------------------------------------------------------|----------------|
|@falkonprods |yes |reviewed PR#9 Discussed/worked on ideas heavily heavy team management |20%|
|@JMKJR |yes |Discussed/worked on ideas heavily |15% | 
|@LNow |yes |Created L3 Clarity coding standards (PR#9) Created stack diagram Drafted contract diagram|15%|
|@pxydn |yes |  |20%|
|Wahlmark|yes|design|15%|

*still adding people*

M0.T1 is due by 29/04/2021. payments should be paid to members before end of that day.

**Comment by cryptopanter** [comment]
Good to have all things archived on github 👍

**Comment by 314159265359879** [comment]
Payments in dollars makes sense, that is also the default for payouts for grants and that way nothing gets lost in conversions.

**Comment by LNow** [comment]
LGTM. 
I prefer BTC/STX over USD

**Comment by JMKJR** [comment]
@314159265359879 is missing on list? If the goal is to reward all who contributed, I believe Werner contributed to our planning phase and even wrote out a small design sketch. Otherwise, all else LGTM. Thanks.

**Reply by Zk2u** [reply]
omg yeah! thank you! this is why i wanted feedback lolll. adding him now :)

**Comment by martinbavio** [comment]
LGTM, although I have no idea how % are being decided.

**Comment by falkonprods** [comment]
I have to let you all know: this grant came to ME directly as taxable income. I am more than happy to share. That said, I will be taxed on it and living in California, and in my bracket that is 12.3% for State, and 35-37% for Federal. Which means LITERALLY HALF OF THIS GRANT MONEY IS GOING TO TAXES. So we don't in actuality have $2500 but more like $1250.  Sorry for the inconvenience but that's how it is. So  I  wouldn't get attached to the amount but rather the percentages. And I intend on paying in crypto because that's way easier.

**Comment by falkonprods** [comment]
Here are  my thoughts on  percentages: PXYDN and I have been driving this project, as such, should get 20%, lnow 15%, JMKJR 15%, Wahlmark 15% (on  completion of his design work), remaining 15% divided up equally among others.

**Comment by falkonprods** [comment]
And future grants should be done via the Labs3 DAO so as to avoid this taxation situation.
